index.html

```diff
@@ -30,20 +30,20 @@ <h1>Privacy Community Group</h1>
 title="Community Group">CG</abbr></a>
 is <a href=charter.html>chartered</a> <q cite=charter.html#mission>to
 incubate privacy-focused web features and APIs to improve user privacy
-on the web through enhanced browser behavior.</q> If that sounds
-interesting to you, please
-<a href=https://www.w3.org/community/privacycg/join>join us</a>!
```

Comment:
Keep this text "if that sounds interesting to you, please join us", but link instead to the participants section (if that's possible).

Reply:
Restored.